cmd/influx/query.go

```diff
@@ -422,7 +422,7 @@ func (f *formatter) valueBuf(i, j int, typ flux.ColType, cr flux.ColReader) []by
 		}
 	case flux.TString:
 		if cr.Strings(j).IsValid(i) {
-			buf = []byte(cr.Strings(j).ValueString(i))
+			buf = []byte(cr.Strings(j).Value(i))
```

Comment:
The CLI code isn't in the `master` branch where this cherry-pick is coming from, so I had to update this method name.